examples/invoker/client.go

```diff
@@ -179,11 +207,16 @@ func startMeasurement(msg string) (string, time.Time) {
 }
 
 func getDuration(msg string, start time.Time) {
-	latency := time.Since(start).Microseconds()
+	latency := time.Since(start)
 	log.Debugf("Invoked %v in %v usec\n", msg, latency)
```

Comment:
I am not sure what the debug print would show. print latency.Microseconds()
```suggestion
	latency := time.Since(start)
	log.Debugf("Invoked %v in %v usec\n", msg, latency.Microseconds())
```